### PR TITLE
Manage keys to suite

### PR DESCRIPTION
**Pull request by Chriserus**
Cherry pick and fix changes regarding managing multiple keys in `Manage Keys` section, activate import keys in `suite` version of the app.

### CODE REVIEW

File: src/views/Home.b.vue [hunk 1]
```diff
@@ -0,0 +1,256 @@
+<template>
```
Comment (aeddaqqa) [inline src/views/Home.b.vue]: delete this file please

File: src/store/types.ts [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 export interface RootState {
     isAuth: boolean
+    storedActiveWallet: null | WalletType
     activeWallet: null | WalletType
     wallets: WalletType[]
     address: String | null
     volatileWallets: WalletType[] // will be forgotten when tab is closed
     warnUpdateKeyfile: boolean
-    theme: 'night' | 'day'
+    walletsDeleted: boolean
```
Comment (aeddaqqa) [inline src/store/types.ts:21]: theme state is missing

**Review by aeddaqqa — COMMENTED** [review]
delete account is not working in this pr

**Comment by Chriserus** [comment]
> delete account is not working in this pr

Fixed in another commit, thanks

File: src/store/types.ts (hunk 2 above)
Comment (Chriserus) [reply src/store/types.ts:21]: This is intentional, Markus removed it

Comment (aeddaqqa) [reply src/store/types.ts:21]: but this leads to this error in the suite app we need to define which of the theme is selected in the suite
Screenshot 2023-03-22 at 14 54 33

**Comment by aeddaqqa** [comment]
remove the account from the manage keys section still does not work
Screenshot 2023-03-22 at 14 58 15

File: src/store/types.ts (hunk 2 above)
Comment (Ayagoumi) [reply src/store/types.ts:21]: @Chriserus this shouldn't be removed.

Comment (Chriserus) [reply src/store/types.ts:21]: Restored

**Review by Ayagoumi — DISMISSED** [review]

**Review by Ayagoumi — APPROVED** [review]

**Pull request merged**